rts/System/Sound/OpenAL/AudioChannel.h

```diff
@@ -24,6 +26,7 @@ class AudioChannel : public IAudioChannel {
 	typedef std::pair<std::string, float> StreamQueueItem;
 
 public:
+	AudioChannel();
```

Comment:
This gets added in this commit and removed in the next one? Also the `= default` is in the `.cpp` file which is somewhat inconsistent with how it looks in other classes.

Reply:
The first idea was to cache `MusicStream` in channel but these changes were reverted.